test/integration/security-rules.spec.ts

```diff
@@ -65,7 +64,32 @@ describe('admin.securityRules', () => {
     return Promise.all(promises);
   }
 
-  after(() => {
+  function getRandomInt(max: number) {
+    return Math.floor(Math.random() * Math.floor(max));
+  }
+
+  /**
+   * Creates a temporary ruleset with a random name that will be deleted upon
+   * completion of the test. (The random name "ensures" that if the ruleset
+   * doesn't get cleaned up for any reason, then it will be unlikely to impact
+   * other tests, including a second run of the same test.)
+   *
+   * @param prefix Optional prefix that will be prefixed to the name of the
+   *   rules file. Useful if your test creates multiple temporary rulesets and
+   *   you want to ensure that they don't collide.
+   */
+  function createTemporaryRuleset(prefix?: string): Promise<admin.securityRules.Ruleset> {
+    const name = (prefix ? prefix : '') + 'random_name_' + getRandomInt(100000) + '.rules';
```

Comment:
This is really not necessary. Rulesets are given a unique name by the Rules API. Rules file names don't have to be globally unique. They just have to be unique within a given ruleset. Since this temp ruleset has only one rules file, we can just give it a fixed name like `firestore.rules`.

Reply:
Oh, I understand now. Done.